src/server/services/wms/qgswmsgetcapabilities.cpp

```diff
@@ -18,6 +18,9 @@
  *   (at your option) any later version.                                   *
  *                                                                         *
  ***************************************************************************/
+
+#include <optional>
```

Comment:
I don't think that this include is needed.

Reply:
No, it's not indeed